src/routes/github/subscription-deferred-install/github-subscription-deferred-install-get.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response } from "express";
+import { extractSubscriptionDeferredInstallPayload } from "services/subscription-deferred-install-service";
+import { hasAdminAccess } from "services/subscription-installation-service";
+
+const escapeHtml = (unsafe: string) => {
```

Comment:
lol, can't we use a lib? 
Or isn't handlerbar already escaping?

Reply:
given that's a throw-away code, I'd leave it as it is

Reply:
I still think handlebar is easier to achieve since we already had it in place. Just throw in a .hbs file and voila.

Reply:
One small vs one new file + call render with params, for temp code?

Reply:
If `escapeHtml` is required, then yes, I'd say trust handlebars ( or some libs -- but cons as one more dependancy) rather than hand crafting the escape, as security related stuff, just don't do it yourself.

Reply:
Well, that's annoying. We are enabling this code only for 1 customer, and this is such a corner-case

Reply:
I'm happy to create some sort of UI with handlebars in my next PR, this is not the purpose of the current one and it is big enough already